.github/workflows/release.yaml

```
@@ -255,10 +250,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       if: ${{ needs.initialize.outputs.release_type == 'minor' }}
       run: |
-        sed -i -r "s/^VERSION \?= (.*)/VERSION \?= $NEXT_VERSION-SNAPSHOT/" Makefile
-
-        jq -r ".version |= \"${NEXT_VERSION:1}\" | .consolePlugin.version |= \"${NEXT_VERSION:1}\"" plugin/package.json > plugin/package.json.tmp
-        mv plugin/package.json.tmp plugin/package.json
+        hack/update-version-string.sh "${NEXT_VERSION}-SNAPSHOT"
 
         git add Makefile plugin/package.json
```

There is a new file that gets updated when we do a release. We have to make sure we add that new file to the release's git commit:

```suggestion
        git add Makefile plugin/package.json plugin/plugin-metadata.ts
```

---

GitHub won't let me add a comment/suggestion to the other line where this change needs to happen, so I need to just document it here. On line 232, add `plugin/plugin-metadata.ts` to the `git add` command:

```
        git add Makefile plugin/package.json
```

should be:
```
        git add Makefile plugin/package.json plugin/plugin-metadata.ts
```